plugins/slo/pkg/apis/slo/slo.proto

```diff
@@ -53,7 +53,7 @@ service SLO {
   }
 
   // Can this metric run with this service & cluster? No == error
-  rpc GetMetric(MetricRequest) returns (google.protobuf.Empty) {
+  rpc AssignMetric(MetricRequest) returns (Metric) {
```

Comment:
the name `AssignMetric` suggests this method will have side effects, but it looks like this is more of a query.

Reply:
Renamed to `GetMetricId`, since we are fetching the exact id of a metric using preconfigured metric name, e.g. `uptime` will try to match the closest metric to prometheus' `up` on a service, etc...